github_connector_odoo/views/view_odoo_license.xml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+Copyright (C) 2016-Today: Odoo Community Association (OCA)
+@author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+<odoo>
+    <record id="view_odoo_license_search" model="ir.ui.view">
+        <field name="model">odoo.license</field>
+        <field name="arch" type="xml">
+            <search>
+                <field name="name" />
+            </search>
+        </field>
+    </record>
+    <record id="view_odoo_license_tree" model="ir.ui.view">
+        <field name="model">odoo.license</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="name" />
+                <field name="module_version_qty" sum="Total" />
+            </tree>
+        </field>
+    </record>
+    <record id="view_odoo_license_form" model="ir.ui.view">
+        <field name="model">odoo.license</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button
+                            name="toggle_active"
```

Comment:
PLease read this... https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#tasks-to-do-in-the-migration
See widget name="web_ribbon"